Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlNavigationCompletedEventArgs.cs

```diff
@@ -10,24 +10,29 @@ namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
     /// <summary>
     /// Provides data for the <see cref="IWebView.NavigationCompleted"/> and <see cref="IWebView.FrameNavigationCompleted"/> events. This class cannot be inherited.
     /// </summary>
-    /// <remarks>Copy from <see cref="Windows.Web.UI.WebViewControlNavigationCompletedEventArgs"/> to avoid requirement to link Windows.winmd</remarks>
+    /// <remarks>Copy from <see cref="global::Windows.Web.UI.WebViewControlNavigationCompletedEventArgs"/> to avoid requirement to link Windows.winmd</remarks>
     /// <seealso cref="System.EventArgs" />
-    /// <seealso cref="Windows.Web.UI.WebViewControlNavigationCompletedEventArgs"/>
+    /// <seealso cref="global::Windows.Web.UI.WebViewControlNavigationCompletedEventArgs"/>
     public sealed class WebViewControlNavigationCompletedEventArgs : EventArgs
     {
-        internal WebViewControlNavigationCompletedEventArgs(Windows.Web.UI.WebViewControlNavigationCompletedEventArgs args)
+        internal WebViewControlNavigationCompletedEventArgs(global::Windows.Web.UI.WebViewControlNavigationCompletedEventArgs args)
         {
             IsSuccess = args.IsSuccess;
             Uri = args.Uri;
             WebErrorStatus = (WebErrorStatus)args.WebErrorStatus;
         }
 
-        internal WebViewControlNavigationCompletedEventArgs(Windows.Web.UI.WebViewControlNavigationCompletedEventArgs args, Uri uri)
+        internal WebViewControlNavigationCompletedEventArgs(global::Windows.Web.UI.WebViewControlNavigationCompletedEventArgs args, Uri uri)
         : this(args)
         {
             Uri = uri;
         }
 
+        internal WebViewControlNavigationCompletedEventArgs(System.Windows.Navigation.NavigationEventArgs e)
```

Comment:
I believe `System.Windows.Navigation` (here and in other common files) is WPF specific and will make spiting up the binary more difficult (trying to do that right now). Is there a way to avoid adding WPF or WinForms specific code outside of the WPF and WinForms folder?

Reply:
If something is specific to WPF/Winforms, then it should be implemented with a partial class

Reply:
See example: e4241694171b347b4877b513f1faf374333dfeae